src/elastic.jl

```diff
@@ -12,14 +12,19 @@ struct ElasticManager <: ClusterManager
     terminated::Set{Int}             # terminated worker ids
     topology::Symbol
     sockname
+    printing_kwargs
 
-    function ElasticManager(;addr=IPv4("127.0.0.1"), port=9009, cookie=nothing, topology=:all_to_all)
+    function ElasticManager(;addr=IPv4("127.0.0.1"), port=9009, cookie=nothing, topology=:all_to_all, printing_kwargs=())
```

Comment:
Seems like there are only two allowed [printing_kwargs here](https://github.com/JuliaParallel/ClusterManagers.jl/pull/136/files#diff-4f952a585791929c8afd59dc90cd061eR146), a tuple seems like a weird way to do this.

Reply:
I just didn't want to balloon the number of keyword arguments for `ElasticManager` and also `ElasticManager(..., printing_kwargs=(same_project=true,))` implies `same_project=true` applies just to printing but `ElasticManager(..., same_project=true)` might not. But I'm happy to change it however you see fit.